pkg/build/node/node.go

```diff
@@ -355,6 +355,12 @@ func (c *BuildContext) prePullImages(dir, containerID string) error {
 		return err
 	}
 
+	// Create "images" subdir.
+	imagesDir := path.Join(dir, "bits", "images")
+	if err := os.Mkdir(imagesDir, 0777); err != nil {
```

Comment:
We should use `MkdirAll` here, which won't fail when the directory already exists (it will for the other build modes)
```suggestion
	if err := os.MkdirAll(imagesDir, 0777); err != nil {
```